### DISCUSSION TITLE
关于翻译工作

### DISCUSSION THREAD

**Discussion by FAN585545** [PCL-Community/PCL2-Language#2] [Q&A] [answered]
我在翻译希腊语的时候，因为中文有的翻译错误，希腊语翻译也有问题，但是中文翻译只能建议不能直接修改，所以我想被授权。谢谢

**Comment by FAN585545** [comment]
![屏幕截图 2024-11-10 111150](https://github.com/user-attachments/assets/413b992d-7af5-46b8-bc19-d3c49964667b)
![屏幕截图 2024-11-10 111216](https://github.com/user-attachments/assets/7b1edc10-9cc0-44ca-a4f5-e53a899706c1)
这个......我已经建议了，希望通过，谢谢

**Comment by tangge233** [comment, marked as answer]
非常抱歉现在才看到这个 Discussion 

这个其实不是错误，是龙腾猫越特意留下的。
这个是程序中读取数据出现编码问题的时候容易出现的一种情况，通常被视为一种错误。这里并非真实存在编码错误，而是用此表示程序出问题了。因此在翻译此段文本的时候，可以认为原文本为“程序出现未知错误”。
